app/views/decidim/consultations/admin/consultations/results.html.erb

```diff
@@ -2,6 +2,9 @@
   <div class="card-divider">
     <h2 class="card-title">
       <%= t "decidim.admin.titles.results" %>
+      <span class="export">
+        <%= link_to t("decidim.admin.consultations.results.export"), decidim_admin_action_delegator.consultation_exports_path(current_consultation), method: :post, class: "button tiny button--title" %>
+      </span>
```

Comment:
This button should be placed before the label below instead but the CSS needs some tweaking for that to look ok. There's no gap between them for instance.